efel/cppcore/LibV1.cpp

```diff
@@ -271,7 +272,7 @@ int LibV1::firing_rate(mapStr2intVec& IntFeatureData,
     return nSize;
   else {
     vector<double> stimStart, stimEnd, peakVTime, firing_rate;
-    double lastAPTime;
+    double lastAPTime = 0.;
```

Comment:
But if we have stimStart[0] == lastAPTime == 0.0 the if statement you added right above would have caught that situation, no ?